extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQControllerTask.java

```diff
@@ -147,6 +149,24 @@ public String getType()
     return TYPE;
   }
 
+  @Override
+  public String getLabel() {
```

Comment:
Could @cryptoe @adarshsanjeev please review the proposed labels for MSQ tasks? I want to ensure they accurately reflect the tasks' purposes. Your insights would be greatly appreciated.

Reply:
The values themselves seem to be correct, and should cover the different use cases for MSQ